fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/dto/ReadResultDTO.java

```diff
@@ -54,6 +55,10 @@ public int size() {
     }
 
     public boolean empty() {
-        return resources.isEmpty();
+        return resources.isEmpty() || chunk;
+    }
+
+    public void makeChunk() {
```

Comment:
I don't understand what this means or when/why it would be called.  Maybe some javadoc is in order?